packages/cli/bin/ceramic.js

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env node
 
-require('../lib/bin/ceramic')
+await import('../lib/bin/ceramic.js')
```

Comment:
woah.  What does `await import` do and when is it necessary?  Why do we use this here but no where else in this PR?

Reply:
You can not `require` an ESM module, that is why `import`. 

You could do `import * as something from '../path.js'`, but the next person reading this would ask what `something`, is and why it is needed here. You could disregard `await` and just do `import`, this would do the job mostly. When something wrong happens though, `await import` reports a path to this particular js file, which is useful in some situations.